UPDATING.md

```diff
@@ -23,6 +23,9 @@ assists people when migrating to a new version.
 
 ## Next
 
+* [9825](https://github.com/apache/incubator-superset/pull/9825):  Add Excel sheet upload
```

Comment:
Perhaps add a slightly more verbose comment to say how it can be enabled, like "Support for Excel sheet upload added. To enable support, install Superset with the optional dependency `excel`."